### ISSUE TITLE
[Npcap] Why is it always harder ?

### ISSUE THREAD

**Issue by gpotter2** [nmap/nmap#1092]
Hello all,

We are developing a cross-platform software that rely on a Winpcap/Npcap installation, when running on windows: https://github.com/secdev/scapy

Because this is a public repo, and that we are civilized people, we added some automatic regression tests, ran on Travis CI for our linux and OSX builds, and on Appveyor for the windows ones ([link](https://ci.appveyor.com/project/secdev/scapy))

This means that on our Windows builds, we're testing `scapy`, which is a networking tool (sniff and many other things), against tons of cases and samples. We're also performing networking tests. This means we need to install `npcap` on the Appveyor builds in an automatic way, so that we can test scapy.

This used to be easy, with `chocolatey` support. However, it was removed by the Npcap team for an ambiguous reason: https://github.com/nmap/nmap/issues/900. Testing the software has suddently became harder, but we used a workaround script that would have installed npcap on our builds:
https://github.com/secdev/scapy/blob/master/.appveyor/InstallNpcap.ps1

Then i learn in 0.97 that silent install has now been removed from further Npcap builds, and that we would not be able to use that script with newer Npcap builds anymore. This means that for now, our builds are stuck running npcap 0.96 and won't be able to test scapy against newer npcap versions.

Also, silent install suddently stop beeing free, with the new "Npcap OEM" versions. This means that our open-source software should now be paying fee to use Npcap....

We have been one of the first software to support Npcap rather than Winpcap, and we have even been encouraging our users to switch to Npcap:
https://github.com/secdev/scapy/blob/master/scapy/arch/pcapdnet.py#L53

This new update simply makes our tests harder, everything more complicated, and really sucks.
I don't think it was the goal of Npcap, which firstly seemed to be a great alternative to Winpcap and was developped `under Google Summer of Code 2013 and 2015` to become a such unpratical tool, based on licensing. Why would we keep Npcap if it's becoming harder and harder to implement ?

I hope I'll get an answer soon... Thanks for listening

**Comment by fyodor** [comment]
Hi @gpotter2.  We are always trying to find the best way forward with Npcap.  I invited you more than six months ago to email me if you wanted to be part of the discussion (https://github.com/nmap/nmap/issues/900#issuecomment-307311651), but you declined.  Instead you just post occasional rants to our bug tracker. And we never asked you to be "paying fee to use Npcap".  We haven't and wouldn't accept any money from you or the Scapy project. Regarding the summer GSoC intern Yang, all the GSoC work he did in 2013 and 2015 was done under the BSD-style WinPcap license, so you and anyone else can use it in pretty much any way you wish.  Current Npcap is a very different product from that.  We just released version 0.98 this week and are preparing for the big 1.0 release soon.  Cheers.
